rrbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_1
    pid: {p: 2, i: 0.0, d: 2}
    has_velocity_limits: true
    max_velocity: 0.05
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_2
    pid: {p: 2, i: 0.0, d: 2}
    has_velocity_limits: true
    max_velocity: 0.05
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_3
    pid: {p: 2, i: 0.0, d: 2}
    has_velocity_limits: true
    max_velocity: 0.05
  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_4
    pid: {p: 2, i: 0.0, d: 2}
    has_velocity_limits: true
    max_velocity: 0.05
  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: prisjoint_1
    pid: {p: 10000, i: 0, d: 100}
    has_velocity_limits: true
    max_velocity: 0.00055
  joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: prisjoint_2
    pid: {p: 10000, i: 0, d: 100}
    has_velocity_limits: true
    max_velocity: 0.0005
 
